### DISCUSSION TITLE
C#: how to target .NET Core, Standart and 5?

### DISCUSSION THREAD

**Discussion by Sarund9** [premake/premake-core#1666] [Q&A] [answered]
The docs don't say anything on how to target any .NET Runtime other than Framework.
According to [this commit](https://github.com/premake/premake-core/pull/1386), Core and Standart are Supported

**Comment by starkos** [comment, marked as answer]
You're right, the docs do need updating, a pull request would be welcomed. Looking at the unit tests provided in the commit, it looks like you do it like this:

```lua
dotnetframework "netcoreapp2.2"
dotnetframework "netstandard1.2"
dotnetframework "4.7.2"
```

_Updated; hit submit too soon…_ I'm guessing that you can find more identifiers by manually editing a project in Visual Studio and seeing what it writes?
